StRoot/StJetMaker/mudst/StjMCMuDst.cxx

```diff
@@ -81,7 +83,45 @@ StjMCParticleList StjMCMuDst::getMCParticleList()
 	}
       }
     }
-  }
+  }else if(genEvent){
+      
+      LOG_INFO <<"There is no geant in _maker. Proceed with Particle info from StarGenEventReader. " << endm;
+      
+      StarGenParticle* particle = 0;
+      
+      int nparts = genEvent->GetNumberOfParticles();
+      
+      int runNumber   = -1;
+      int eventNumber = -1;
 
+      runNumber = genEvent->GetRunNumber();
+      eventNumber = genEvent->GetEventNumber();
+      
```

Comment:
Hi Manny, 

Do you know if the StarGenEvent saved the mstu72 and mstu73 information? Those two parameters are important because they delimits the parton records from Pythia 6. You can refer the lines above in this code:

StjMCParticleCutParton::mstu72 = pythiaTable->mstu72;
StjMCParticleCutParton::mstu73 = pythiaTable->mstu73;